src/wallet/wallet.cpp

```diff
@@ -3922,6 +3927,7 @@ std::string CWallet::GetWalletHelpString(bool showDebug)
     {
         strUsage += HelpMessageGroup(_("Wallet debugging/testing options:"));
 
+        strUsage += HelpMessageOpt("-bypasskeypoolcritical", _("Bypass keypool critical limit. Don't shutdown the node if keypool drops below keypoolcritical limit (but don't advance BestBlock)"));
```

Comment:
Oh, guess the doc is right, but just super confusing (double negative-ish, but not really).Maybe "but *still* don't advance"?